app/exerlog/lib/UI/calendar/calendar_page.dart

```diff
@@ -1,7 +1,10 @@
 import 'package:exerlog/UI/global.dart';
 import 'package:exerlog/UI/gradient_button.dart';
 import 'package:exerlog/UI/workout/workout_page.dart';
+import 'package:exerlog/main.dart';
+import 'package:firebase_auth/firebase_auth.dart';
 import 'package:flutter/material.dart';
+
 import 'calendar_widget.dart';
 
 class CalendarPage extends StatefulWidget {

```

Comment:
Please change this widget from a StatefulWidget into a StatelessWidget. This widget isn't using a setState and therefore won't change its state.
```suggestion
class CalendarPage extends StatelessWidget {

```